target/linux/ipq40xx/image/Makefile

```diff
@@ -216,6 +216,22 @@ define Device/compex_wpj428
 endef
 TARGET_DEVICES += compex_wpj428
 
+define Device/corewav_m018u0-s1
+	DEVICE_VENDOR := Corewav
+	DEVICE_MODEL := M018U0
+	DEVICE_VARIANT := S1
+	BOARD_NAME := m018u0-s1
+	DEVICE_DTS := qcom-ipq4018-corewav-m018u0-s1
```

Comment:
Please have a look at recently merged commit 9c7025b7794ae06774632499547f78d827d13499.
Implications for you:
- Rename DTS(I) files by removing "corewav-"
- Remove DEVICE_DTS here and add `SOC := qcom-ipq4018`
- Update 901-arm-boot-add-dts-files.patch and DTS include lines.

Reply:
Thanks for your review. I changed these files

Reply:
Looks good as far as these changes are concerned.